### PR TITLE
Feature - IJ-187 Update Team Member Edit

### PR DESCRIPTION
**Pull request by garethdandrews**
Updated the Team Member Edit view to mirror the changes made to the Users Edit view. These include: 
- dividing the form up into subheadings 
- JavaScript to hide the second change password field until the first is clicked
-  extracting the confirm password to a modal

Screenshot from 2021-05-26 11-05-53
Screenshot from 2021-05-26 11-07-22

Additionally in this PR, I have added user pronouns to the User Index and Show views:

Screenshot from 2021-05-26 11-09-10
Screenshot from 2021-05-26 11-09-29

### CODE REVIEW

File: app/assets/stylesheets/users.scss [hunk 1]
```diff
@@ -19,8 +19,15 @@
...
+      small {
+        color: $dark-grey;
+      }
+
       &:hover {
         color: $white;
+        small {
+          color: $white;
+        }
       }
```
Comment (garethdandrews) [inline app/assets/stylesheets/users.scss:31]: My SCSS isn't that great yet but is there a cleaner way to write this?

Comment (phil-6) [reply app/assets/stylesheets/users.scss:31]: There's a lot of options, I'd probably keep the small grouped together.
``` scss
&:hover {
    color: $white;
}
small {
    color: $dark-grey;
    &:hover {
        color: $white;
    }
}

```

File: app/views/team_members/registrations/edit.html.erb [hunk 2]
```diff
@@ -1,70 +1,89 @@
...
+          </button>
+        </div>
 
-      <div class="form-floating">
-        <%= f.email_field :email, class: 'form-control', autocomplete: "email", placeholder: "Email Address" %>
-        <%= f.label :email %>
-      </div>
+        <%= link_to "Cancel my account", registration_path(resource_name), data: { confirm: "Are you sure?" }, method: :delete, class: "btn btn-danger" %>
+      
+        <%= render 'modals/confirm_edit', f: f %>
```
Comment (phil-6) [inline app/views/team_members/registrations/edit.html.erb:82]: "f" could probably be more descriptive. We don't lose anything by using full words. This isn't matlab! :wink:

**Review by phil-6 — APPROVED** [review]
Looks good to me, great work Gareth.

**Pull request merged**